consensus/clique/src/main/java/org/hyperledger/besu/consensus/clique/CliqueMiningTracker.java

```diff
@@ -42,4 +42,8 @@ public boolean canMakeBlockNextRound(final BlockHeader header) {
   public boolean blockCreatedLocally(final BlockHeader header) {
     return CliqueHelpers.getProposerOfBlock(header).equals(localAddress);
   }
+
+  public ProtocolContext getProtocolContext() {
```

Comment:
this shouldn't be exposed here - this class isn't a carriage for this - its used internally to determine if/how we can mine.

Reply:
This part has been moved to the mining tracker class. This is no longer exposed